### PR TITLE
aur-build: propagate --pacman-conf to makepkg for local builds

### PR DESCRIPTION
**Pull request by AladW**
Noticed this as my package tests would not use their custom set cache directory: `makepkg -s` runs `pacman` with a few hardcoded arguments; in particular, `--config` is not supported. The only way around this is either use a chroot, or set `PACMAN` to a `pacman --config` wrapper. As former is already possible, this PR implements latter.

### CODE REVIEW

File: lib/aur-build [hunk 1]
```diff
@@ -222,11 +225,24 @@ if (( chroot )); then
     printf 'Running aur-chroot %s\n' >&2 "${args[*]}"
     aur chroot "${args[@]}"
 else
+    # makepkg -s does not support setting pacman.conf (#)
```
Comment (maximbaz) [inline lib/aur-build:228]: Did you want to link to a github issue here?

**Review by maximbaz — APPROVED** [review]
Looks good, just one typo

File: lib/aur-build (hunk 1 above)
Comment (AladW) [reply lib/aur-build]: I thought of filing an issue first, but I went straight to the PR. Removed the #

**Comment by AladW** [comment]
Thanks 👍

**Pull request merged**